src/Bicep.Core.UnitTests/Diagnostics/LinterRuleTests/UseRecentApiVersionRuleTests.cs

```diff
@@ -756,11 +756,10 @@ private static IEnumerable<object[]> GetTestData()
                 {
```

Comment:
I think it would be fair to disable this for test code if you wanted.

Reply:
Yeah that's fair. It's definitely less critical to adhere strictly to the rule when writing tests, and sometimes it may get in the way. Added .editorconfig files to test projects to override the Severity to suggestion.